BrainPortal/cbrain_plugins/cbrain-plugins-base/userfiles/csv_file/csv_file.rb

```diff
@@ -89,26 +89,26 @@ def self.guess_csv_quote_sep(path)
 
     quote_char_list.each do |qc|
       col_sep_list.each  do |cs|
-        combinaison_key = [qc,cs]
+        combinations_key = [qc,cs]
```

Comment:
Like I said in my previous code review, please do not adjust the code in this file, except for the very last change (at the word `useful`).